plugins/database/redshift/redshift_test.go

```diff
@@ -189,71 +206,103 @@ func TestPostgreSQL_RenewUser(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 
-	statements := dbplugin.Statements{
-		Creation: []string{testRedshiftRole},
-	}
-
-	usernameConfig := dbplugin.UsernameConfig{
+	usernameConfig := dbplugin.UsernameMetadata{
 		DisplayName: "test",
 		RoleName:    "test",
 	}
 
-	username, password, err := db.CreateUser(context.Background(), statements, usernameConfig, time.Now().Add(2*time.Second))
+	const password = "SuperSecurePa55w0rd!"
+	newReq := dbplugin.NewUserRequest{
+		UsernameConfig: usernameConfig,
+		Password:       password,
+		Statements:     dbplugin.Statements{Commands: []string{testRedshiftRole}},
+		Expiration:     time.Now().Add(5 * time.Second),
```

Comment:
Can this expiration be smaller since we're doing a sleep with it later?

Reply:
It was 2 seconds before, but I was worried that that felt quite short and could lead to flakiness. Happy to revert it back to 2s if you think that is a more reasonable balance of risk.

Reply:
Since this is hitting an actual redshift instance in AWS I think it's reasonable to have this larger. I didn't realize this wasn't hitting a local service previously.

Reply:
To be fair, my increase to the timings wasn't based on any data. Even with the Redshift DB based in `us-east-1`, the tests still reliably pass from my machine as they are now.